pkg/netpol/connlist/connlist.go

```diff
@@ -113,10 +113,18 @@ func FromK8sCluster(clientset *kubernetes.Clientset) ([]Peer2PeerConnection, err
 // getConnectionsList returns connections list from PolicyEngine object
 func getConnectionsList(pe *eval.PolicyEngine) ([]Peer2PeerConnection, error) {
 	res := []Peer2PeerConnection{}
```

Comment:
- is `Peer2PeerConnection` used by other packages and should remain public?
- given the package name `connlist`, I wonder if `Peer2PeerConnection` is not too long of a name and whether we have the right abstraction level. Need to think about it, not required in this PR